tensorflow/contrib/layers/python/layers/initializers.py

```diff
@@ -46,7 +46,7 @@ def xavier_initializer(uniform=True, seed=None, dtype=dtypes.float32):
   Args:
     uniform: Whether to use uniform or normal distributed random initialization.
     seed: A Python integer. Used to create random seeds. See
-      [`set_random_seed`](../../api_docs/python/constant_op.md#set_random_seed)
+      [`set_random_seed`](../../../../../api_guides/python/constant_op#random-tensors)
```

Comment:
Sorry, I am not fully sure why this is the case. One of the tool you guys are using will interpolate that token with actual values when building tensorflow.org?

If so, you want me to change be like below, right?
```py
[`set_random_seed`](${$constant_op#random-tensors})
```